sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - [Breaking] Aspects in opinions mining are now called targets and each individual opinion is now called an assessment. The new naming simplifies the naming of different parts of the response.
 - `beginAnalyzeBatchActions` can now process recognize linked entities actions.
 - `beginAnalyzeHealthcareEntities` entities now include `assertions` instead of `isNegated` which gives more context about the respective entity.
+- `recognizePiiEntities` takes a new option, `piiCategories`, that specifies a list of Pii categories to return.
```

Comment:
Consider adding a sample that shows the usage of this filter

Reply:
Done.

Reply:
should we consider to rename to `categoriesFilter`

Reply:
Of course we have to :D, good catch. Fixed.